5-Heroku/server/todo_service.js

```diff
@@ -0,0 +1,94 @@
+const mongoose = require("mongoose");
+const uuid = require("uuid");
+require('dotenv').config();
+
+const Schema = mongoose.Schema;
+
+const MONGO_URI = process.env.MONGO_URI;
+
+mongoose.connect(MONGO_URI, {
+  useNewUrlParser: true,
+  useUnifiedTopology: true,
+});
+
+mongoose.connection.on("connected", () => {
+  console.log("MobgoDB is connected!");
+});
+
+const TodoSchema = new Schema({
+  user_id: String,
+  id: String,
+  text: String,
+  check: Boolean,
+});
+
+const Todo = mongoose.model("Todo", TodoSchema);
+
+function dbUpdateCallback(req, res, successStatus) {
+  return function (error, doc) {
+    if (error) {
+      console.log(error);
+      res.status(500).send("Database error!");
+    } else {
+      res.sendStatus(successStatus);
+    }
+  };
+}
+
+module.exports.getItems = (req, res) => {
+  let userId;
+
+  if (req.cookies === undefined ||
+    req.cookies.userId === undefined) {
+    userId = uuid.v4();
+    res.cookie('userId', userId, { maxAge: 900000 });
```

Comment:
the `req` object is well an object :)
so once u read the userId you can write something like
`req.userId = userId` 
and then everywhere in the code that u want to get it from it will be nicer then `req.cookies.userId`